packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCPD ITI-55 Processor/destinationConnector-XCPD Endpoint-responseTransformer-step-5-Case 5: Application Error.js

```diff
@@ -24,7 +24,9 @@ if (['AE','AR'].indexOf(ack.toString()) > -1 || ['AE','QE'].indexOf(queryRespons
 	
 	for each (var trigger in xml.*::['controlActProcess'].*::['reasonOf'].*::['detectedIssueEvent'].*::['triggerFor']) {
 		
-		reason += trigger.*::['actOrderRequired'].*::['code']['@code'].toString() + ' ';
+
+		const reason = trigger.*::['actOrderRequired'].*::['code']['@code'].toString() + ' ';
+    channelMap.put('REASON_WARNING', reason);	
```

Comment:
1. `reason`'s source is the same as `issue.details.text` below
   - maybe we remove that space at the end and use it there too?
3. This is running inside a loop, so a second iteration will remove the first value on `REASON_WARNING`
   - maybe instead add a `channelMap.put('OPERATION_OUTCOME', operationOutcome);` at line 38?